browse/routes/ui.py

```diff
@@ -16,8 +17,7 @@
 def before_request() -> None:
     """Get instituional affiliation from session."""
     if 'institution' not in session:
-        institution = get_institution_from_request()
-        session['institution'] = institution
+        session['institution'] = get_institution(request.remote_addr)
```

Comment:
This change seems OK--more straightforward. I wonder if this should be handling exceptions though?

Reply:
Goes to arxiv-base 500. It's less code and doesn't need a test. 

get_institution() might have to go arxiv-base if other pages need that institutional banner too.